sys/include/atomic_utils.h

```diff
@@ -6,6 +6,8 @@
  * details.
  */
 
+ #pragma once
```

Comment:
```suggestion
#pragma once
```

That fixes the failing static test.

Reply:
I wonder... since this has happened more than once now (not necessarily to you, but in general) this might motivate adding that case to the headerguard Python script 🤔

Reply:
I honestly wonder why my linter did not complain on a preprocessor directive with leading space. To my knowlege the following would not be correct with some strict interpretation of the C standard:

```C
#ifndef FOO
  #define FOO 42
#endif
```

which is why we use

```C
#ifndef FOO
#  define FOO 42
#endif
```

If you intend to spend time on it, maybe just flag known preprocessor directives with leading space before the `#` instead to also catch other cases of incorrect use?

Reply:
This would probably require a separate static test, since the `whitespace` test uses a feature of git that highlights "invalid" trailing whitespaces:

https://github.com/RIOT-OS/RIOT/blob/d9938cc634c1bfd125e13ca6d300edfc5078f0e2/dist/tools/whitespacecheck/check.sh#L43-L47

The `#pragma once` leading whitespace check can be easily added to the `headerguards` Python script (in fact I already did, just have to create a PR).